metabase/metabase
# [WIP] Add search loading indicator to QB filters

Fixes https://github.com/metabase/metabase/issues/37955
Demo https://www.loom.com/share/0932accf1df84da1ad19f0fb1f8e6f50

Adds a loading indicator to `SearchValuePicker`. The loader works correctly with caching & frontend-side (subsequent) search.

How to verify:
- Open People
- Click Email -> Filter by this column
- Change operator to `Is` (we will fix this soon and `Is` would be the default value)
- Try typing people names
- Check that the loader and "nothing found" message make sense
- Try disabling network and check that we handle errors, too

Tests TBD.

<img width="322" alt="Screenshot 2024-07-10 at 22 42 33" src="https://github.com/metabase/metabase/assets/8542534/22392a6c-868b-46fa-8d80-27e3909f908c">
<img width="325" alt="Screenshot 2024-07-10 at 22 42 37" src="https://github.com/metabase/metabase/assets/8542534/945e905a-b595-4478-85ce-f8a56b354fea">
<img width="363" alt="Screenshot 2024-07-10 at 23 09 10" src="https://github.com/metabase/metabase/assets/8542534/a114fdfe-6981-43e0-b51c-e4de1766d4ae">
